src/core/tokenHandler.js

```diff
@@ -106,64 +107,58 @@ export default function tokenHandlerFactory(options) {
          * @param {String} newToken
          * @returns {Promise<Boolean>} - resolves true if successful
          */
-        setToken: function setToken(newToken) {
+        setToken(newToken) {
             return tokenStore.enqueue(newToken);
         },
 
         /**
          * Extracts tokens from the Client Config which should be received on every page load
          * @returns {Promise<Boolean>} - resolves true when completed
          */
-        getClientConfigTokens: function getClientConfigTokens() {
-            var self = this;
-            var clientTokens = _.map(module.config().tokens, function(serverToken) {
-                return {
-                    value: serverToken,
-                    receivedAt: Date.now()
-                };
-            });
+        getClientConfigTokens() {
+            const clientTokens = _.map(module.config().tokens, serverToken => ({
```

Comment:
We can simply use map function form Array to avoid using lodash, if you like :)

Reply:
Yep, this is the kind of use case where I tend to prefer to use lodash, as we cannot rely on the data. But anyway, I managed the replacement ;)